src/App.jsx

```diff
@@ -1,104 +1,22 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+const postsWithData = postsFromServer.map(post => {
```

Comment:
The better way is to divide into separate functions 
example:  `getUsersById` and `getComentsByUser`

Reply:
Good point, fix it

Reply:
done